state/core_access.go

```diff
@@ -118,8 +120,8 @@ func NewCoreAccessor(
 func (ca *CoreAccessor) Start(ctx context.Context) error {
 	ca.ctx, ca.cancel = context.WithCancel(context.Background())
 	// create the staking query client
-	ca.stakingCli = stakingtypes.NewQueryClient(ca.coreConn)
-	ca.feeGrantCli = feegrant.NewQueryClient(ca.coreConn)
+	ca.stakingCli = stakingtypes.NewQueryClient(ca.appConn)
+	ca.feeGrantCli = feegrant.NewQueryClient(ca.appConn)
```

Comment:
so these don't work with the coreConn?

Reply:
the tests pass with both, but actually I don't see any usages of the calling functions that use these underlying clis. I will add another TODO here to verify this with the node team.